docs/batch_inference_with_ts.md

```diff
@@ -30,27 +31,49 @@ TorchServe's default handlers support batch inference out of box except for `tex
 
 To support batch inference, TorchServe needs the following:
 
-1. TorchServe model configuration: Configure `batch_size` and `max_batch_delay` by using the  "POST /models" management API.
+1. TorchServe model configuration: Configure `batch_size` and `max_batch_delay` by using the  "POST /models" management API or settings in config.properties.
    TorchServe needs to know the maximum batch size that the model can handle and the maximum time that TorchServe should wait to fill each batch request.
 2. Model handler code: TorchServe requires the Model handler to handle batch inference requests.
 
 For a full working example of a custom model handler with batch processing, see [Hugging face transformer generalised handler](https://github.com/pytorch/serve/blob/master/examples/Huggingface_Transformers/Transformer_handler_generalized.py)
 
 ### TorchServe Model Configuration
 
-To configure TorchServe to use the batching feature, provide the batch configuration information through [**POST /models** API](management_api.md).
+Started from Torchserve 0.4.1, there are two methods to configure TorchServe to use the batching feature:
+1. provide the batch configuration information through [**POST /models** API](management_api.md).
+2. provide the batch configuration information through configuration file, config.properties.
 
 The configuration that we are interested in is the following:
```

Comment:
The configuration properties we're interested in are the following

Configuration sounds like the file